.ci/travis/travis.sh

```diff
@@ -264,6 +264,32 @@ no-error-test-sbe)
   removeFolderWithProtectedFiles simple-binary-encoding
   ;;
 
+no-error-xwiki)
```

Comment:
`travis.yml` has to be modified to call this new option.

Reply:
Something like     # No error testing - simple-binary-encoding

Reply:
Following no-error-test-sbe's example I added it to `.drone.yaml` (but I'm not sure if it's fine in pipeline 5).